Add vulnerability exploiters' explanation pages (#4247)
guardicore/monkey: docs/content/features/exploiters/_index.md

Reviewer: Can we try and mirror the structure in `features/credentials_collectors/_index.md`?

Reviewer: We should perhaps make some mention that some exploiters rely on vulnerabilities while others use brute-force tactics to gain access.

---
title: "Exploiters"
chapter: true
pre: "<i class='fas fa-ethernet'></i> "
---

# Exploiters

##

> In a network breach, the exploitation phase is a critical stage where attackers

Reviewer: ```suggestion
During an attack, the exploitation phase is a critical stage where attackers
```

> In a network breach, the exploitation phase is a critical stage where attackers

Reviewer: IMHO this is an unnecessary intensifier. See https://www.wordrake.com/blog/delete-intensifiers-and-qualifiers or https://www.elegantlegalwriting.com/p/delete-unnecessary-modifiers-especially for more opinions on the subject.

```suggestion
In a network breach, the exploitation phase is when attackers
```

identify specific vulnerabilities in the environment through reconnaissance and

> scanning, and accordingly select appropriate exploitation techniques to

Reviewer: ```suggestion
scanning and, accordingly, select appropriate exploitation techniques to
```

But this sentence may be a bit long. Perhaps break it up into two or more.

compromise the network. By leveraging these vulnerabilities, attackers are able
to gain unauthorized access to systems, escalate privileges, or execute
malicious actions.

Infection Monkey has several exploiters that mimic real-world attack vectors.
To the best of our knowledge, most of these exploiters pose no risk to
performance or services on victim machines. The potentially harmful ones are

> clearly marked as "unsafe" in the Monkey Island.

Reviewer: Maybe instead of saying "potentially harmful" we should say "Exploiters that could adversely impact production workloads" or something similar. I realize it's much more verbose, but we don't want to give the impression that Infection Monkey has malicious intent.

---
title: "Exploiters"
chapter: true
pre: "<i class='fas fa-ethernet'></i> "
---

# Exploiters

Exploiters are Infection Monkey plugins that attempt to infect machines
throughout the network by spreading copies of the Agent.

## Mimicking attackers

> During an attack, the exploitation phase is when attackers carry out chosen

Reviewer: This first sentence seems really long.
